operator/src/main/java/org/keycloak/operator/Config.java

```diff
@@ -19,12 +19,15 @@
 
 import io.smallrye.config.ConfigMapping;
 
+import java.util.Optional;
+
 /**
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
  */
 @ConfigMapping(prefix = "operator")
 public interface Config {
     Keycloak keycloak();
+    Optional<String> deployment();
```

Comment:
This does not belong here as it is a test config. As agreed offline, please use a different config prefix (e.g. `tests.operator`) for tests.